includes/shortcodes.php

```diff
@@ -118,7 +118,7 @@ function tsml_ui()
 	));
 
 	// use meetings.json if it's writable, otherwise use the admin-ajax URL to the feed
-	$data = $tsml_cache_writable && file_exists(WP_CONTENT_DIR . $tsml_cache) && defined('ABSPATH') ? get_site_url() . str_replace(ABSPATH, '/', WP_CONTENT_DIR) . $tsml_cache : admin_url('admin-ajax.php') . '?action=meetings&nonce=' . wp_create_nonce($tsml_nonce);
+	$data = $tsml_cache_writable && file_exists(WP_CONTENT_DIR . $tsml_cache) && defined('ABSPATH') ? get_site_url() . str_replace(ABSPATH, '/', WP_CONTENT_DIR) . $tsml_cache  . '?' . filemtime(WP_CONTENT_DIR . $tsml_cache) : admin_url('admin-ajax.php') . '?action=meetings&nonce=' . wp_create_nonce($tsml_nonce);
```

Comment:
could this be simplified with `content_url`?

```
	$data = $tsml_cache_writable && file_exists(WP_CONTENT_DIR . $tsml_cache) ?
		content_url($tsml_cache)  . '?' . filemtime(WP_CONTENT_DIR . $tsml_cache) :
		admin_url('admin-ajax.php') . '?action=meetings&nonce=' . wp_create_nonce($tsml_nonce);
```

Reply:
(i realize that's not related to this specific change, which looks good)

Reply:
I changed this in PR #769 because of this issue #754. However, would be happy to consider other solutions for that issue. I don't have any experience with Cross-Origin Request Blocked that you mention in https://github.com/code4recovery/12-step-meeting-list/issues/754#issuecomment-1094406676

Reply:
oh! sounds like he solved the issue by turning off the caching, do you know if this would work? 

my guess would be that you’d need to get rid of `get_site_url()` but i couldn’t say for sure

Reply:
anyway resolving this conversation this conversation since that's already in `main`